go/dhcp/README.md

```diff
@@ -38,20 +45,20 @@ curl http://127.0.0.1:22222/api/v1/dhcp/stats/eth1.137 | python -m json.tool
 
 ```
    "192.168.0.0/24": {
-        "Category": "registration",
-        "Free": 253,
-        "Interface": "eth1.137",
-        "Members": {
+        "category": "registration",
+        "free": 253,
+        "interface": "eth1.137",
+        "members": {
             "10:1f:74:b2:f6:a5": "192.168.0.2"
         },
-        "Network": "192.168.0.0/24",
-        "Options": {
-            "OptionDomainName": "inlinel2.fabianfence",
-            "OptionDomainNameServer": "10.10.0.1",
-            "OptionIPAddressLeaseTime": "123",
-            "OptionNetBIOSOverTCPIPNameServer": "172.20.135.2",
-            "OptionRouter": "192.168.0.1",
-            "OptionSubnetMask": "255.255.255.0"
+        "network": "192.168.0.0/24",
+        "options": {
+            "optionDomainName": "inlinel2.fabianfence",
```

Comment:
I think it will return OptionDomainName because we use https://github.com/krolaw/dhcp4/blob/master/packet.go

Reply:
See commit https://github.com/inverse-inc/packetfence/pull/2959/commits/e46b342e83d3f5989febcc3d69922d9276363e89

Tested that it returns first-char as lowercase.